app/assets/javascripts/wikis.js

```diff
@@ -12,14 +12,20 @@ function setupWiki(node_id, title, raw, logged_in) {
       wysiwyg: true,
       preProcessor: preProcessMarkdown,
       postProcessor: postProcessContent,
-      extraButtons: {"fa-question": questionForm}
+      extraButtons: { "fa-question": questionForm }
     });
     $('#content').hide();
   } else {
     $('#content').html(shortCodePrompt($('#content')[0], { submitUrl: '/wiki/replace/' + node_id }));
     postProcessContent();
     addDeepLinks($('#content'));
   }
+
+  function questionForm(qbutton, uniqueId) {
+    wiki_title = wiki_title.replace(/ /g, "-");
+    qbutton.attr('href','/questions/new?tags=response:' + node_id + '&question%3A' + wiki_title + ', ' + wiki_title + '-' + uniqueId+', a-wiki-question&template=question&redirect=question').attr('target', '_blank'); 
```

Comment:
@jywarren, I think here `&` symbol in `&question%3A` should be changed into a `,` symbol. Because with this change other tags are not setting correctly